workflow/scripts/sbs_process/eval.py

```diff
@@ -0,0 +1,91 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+
+from lib.sbs_process.eval import (
+    load_and_concatenate_hdfs,
+    plot_mapping_vs_threshold,
+    plot_mapping_vs_threshold,
+    plot_read_mapping_heatmap,
+    plot_cell_mapping_heatmap,
+    plot_reads_per_cell_histogram,
+    plot_gene_symbol_histogram,
+)
+
+# Read barcodes
+df_design = pd.read_csv(snakemake.params.df_design_path, sep="\t")
+df_pool = df_design.query("dialout==[0,1]").drop_duplicates("sgRNA")
+df_pool["prefix"] = df_pool.apply(lambda x: x.sgRNA[: x.prefix_length], axis=1)
+barcodes = df_pool["prefix"]
+
+# Concatenate files
```

Comment:
Does it make sense to leave this in eval? Should this just be part of our combine_* rules? 

The well based combine_* aren't really used for anything...

Reply:
This would also make eval truly an eval script...

Reply:
Yep this makes sense. Done in https://github.com/cheeseman-lab/brieflow/pull/10/commits/c1df54de3a10686c63613494f2bef3c8d1802b5e